config/manifests/bases/vault-config-operator.clusterserviceversion.yaml

```diff
@@ -90,6 +114,11 @@ spec:
       kind: SecretEngineMount
       name: secretenginemounts.redhatcop.redhat.io
       version: v1alpha1
+    - description: VaultSecret is the Schema for the vaultsecrets API
```

Comment:
why is this being added? this should already have been here...

Reply:
Too many rebase :P, apart joke mmm maybe was never committed?